action.yml

```diff
@@ -5,26 +5,34 @@ branding:
   icon: circle
   color: red
 inputs:
-  new-image-name:
-    description: 'Name of the new image that will be created'
+  image:
+    description: 'The name (reference) of the image to build'
     required: true
   base-image:
-    description: 'Base image to use'
+    description: 'The base image to use to create a new container'
```

Comment:
to create a new 'image', not 'container', I believe

Reply:
Technically the word "container" is correct here. You first create a container from scratch, fill it with all you need and then create an image from it.

Reply:
In buildah.io they use the word `container image` maybe we can use that one

Reply:
`Container image` sounds good to me.